odoo/service/server.py

```diff
@@ -388,9 +405,9 @@ def process_limit(self):
             self.limits_reached_threads.add(threading.current_thread())
 
         for thread in threading.enumerate():
-            if not thread.daemon or getattr(thread, 'type', None) == 'cron':
+            if not thread.daemon and getattr(thread, 'type', None) not in ('websocket', 'cron'):
                 # We apply the limits on cron threads and HTTP requests,
```

Comment:
This seems to have changed the meaning of this test entirely? The comment specifically says that we want to apply the limit to cron threads, but the test now makes it *not* apply to cron threads.

Reply:
Indeed, updated to match the expected behavior :+1: